### ISSUE TITLE
Provide a distribution with yunit installed.

### ISSUE THREAD

**Issue by dannywillems** [yunit-io/yunit#74]
It encourages to use it. If no distribution provides it by default, not a lot of users will use/install it.

**Comment by Leppan** [comment]
What would it be based on? What would it be called?

How about "Yubuntu"

**Comment by dannywillems** [comment]
@Leppan I think Ubuntu is a good basis. Yubuntu sounds great.

**Comment by horsemanoffaith** [comment]
Seeing as how you're using Ubuntu as a development platform, that would be good during the development process. I think, however, that the goal of the developers is to make Yunit a stand-alone DE, one that could be used across any distribution.

**Comment by Leppan** [comment]
Can't you use KDE in any distribution, and not just Kubuntu

**Comment by horsemanoffaith** [comment]
Yup, you can. What I meant by that is I don't want to see Yunit tied down to a specific distro. I'd love to see it available for every distro!

**Comment by Leppan** [comment]
Well, we could have Yubuntu, plus just have Yunit available in the software repos and compiles-able from source.

**Comment by WLBI** [comment]
Yubuntu sounds great. Is there already an .iso available? hehe *joke*

**Comment by pixelr0** [comment]
at least maybe a PPA for ubuntu? and the equivalent of PPA for other distros (me noob)

**Comment by pixelr0** [comment]
https://launchpad.net/~ci-train-ppa-service/+archive/ubuntu/stable-phone-overlay

**Comment by adred** [comment]
Yunit OS

**Comment by deriherdianto** [comment]
yOs

**Comment by dannywillems** [comment]
Maybe Yunit OS is better.

**Comment by tenleftfingers** [comment]
It would be nice to become an officially recognised flavour. I realise there is a distance to travel first, but the criteria are here and it would be good to be cognisant of them from the outset https://wiki.ubuntu.com/RecognizedFlavors

**Comment by jsalatas** [comment]
I hope that you understand that we have a loooooonngggggggg way until we reach there :)

**Comment by tenleftfingers** [comment]
Yes :) But I saw infrastructural requirements that I thought were worth bringing up sooner than later.

**Comment by Allanitomwesh** [comment]
I'd like to contribute to this topic.
I think the best thing to do is to become an official Debain DE. They have the strictest policies on freedom and openness and would pretty much guarantee a fully open source project.
